scene/gui/text_edit.cpp

```diff
@@ -3731,6 +3862,8 @@ void TextEdit::_update_caches() {
 	cache.line_spacing = get_constant("line_spacing");
 	cache.row_height = cache.font->get_height() + cache.line_spacing;
 	cache.tab_icon = get_icon("tab");
+	cache.folded_icon = get_icon("Collapse", "EditorIcons");
+	cache.can_fold_icon = get_icon("Forward", "EditorIcons");
```

Comment:
Could probably create new icons for code folding, or at least keep them separate by adding them [here](https://github.com/godotengine/godot/blob/master/editor/editor_themes.cpp#L792)